### PR TITLE
fix(service-list): filter service name avatar resize

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

**Before**
Screenshot 2024-09-02 at 18 49 43

**Now**
Screenshot 2024-09-02 at 18 50 35

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Help yourself at 2AM by putting some metrics to help observability

File: libs/domains/services/feature/src/lib/service-avatar/service-avatar.tsx [hunk 1]
```diff
@@ -25,9 +25,16 @@ export const ServiceAvatar = forwardRef<ElementRef<typeof Avatar>, ServiceAvatar
   return (
     <Avatar
       ref={ref}
+      className={className}
```
Comment (ctjhoa) [inline libs/domains/services/feature/src/lib/service-avatar/service-avatar.tsx:28]: No need to add this as remaining `props` are already spread line 42

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: libs/domains/services/feature/src/lib/service-avatar/service-avatar.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-avatar/service-avatar.tsx]: Yes my bad https://github.com/Qovery/console/pull/1630/commits/e592c870cdf2af9eb23054ca396732f4f16a36f5

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**